responsibleai/requirements.txt

```diff
@@ -1,4 +1,4 @@
-dice-ml>=0.6.1,<0.7
+dice-ml>=0.7.1
```

Comment:
should we also assign an upper bound?
major version upgrade may end up with breaking changes.

Reply:
Good point, Let me upper bound it at 0.8.